main.go

```diff
@@ -1,9 +1,29 @@
 package main
 
 import (
+	"context"
 	"fmt"
+
+	"github.com/platzily/email-consumer/config"
+	"github.com/platzily/email-consumer/drivers/mongodb"
 )
 
 func main() {
-	fmt.Printf("This is the main consumer")
+
+	mongoConfig := config.ReadMongoDBConfig()
+
+	client := mongodb.NewConnection(mongoConfig.URL)
+	defer client.Disconnect(context.TODO())
+
+	eventRepository := mongodb.New(client)
+
+	result, _ := eventRepository.GetById(0)
+	fmt.Printf("This is the result: %v", result)
+
+	eventRepository.UpdateById(0, "Updated")
+
+	updatedResult, _ := eventRepository.GetById(0)
+	fmt.Printf("This is the updated result: %v", updatedResult)
+
+	fmt.Println("Hello world, from notification email consumer")
```

Comment:
In the future, we have to pass this code to a tests suite since this is not the right place for this.